instrumentation/khttp/src/main/java/com/splunk/opentelemetry/instrumentation/khttp/KHttpSingletons.java

```diff
@@ -45,7 +45,8 @@ public final class KHttpSingletons {
                 INSTRUMENTATION_NAME,
                 HttpSpanNameExtractor.create(httpAttributesGetter))
             .setSpanStatusExtractor(HttpSpanStatusExtractor.create(httpAttributesGetter))
-            .addAttributesExtractor(HttpClientAttributesExtractor.create(httpAttributesGetter))
+            .addAttributesExtractor(
+                HttpClientAttributesExtractor.create(httpAttributesGetter, netAttributesGetter))
```

Comment:
This is unrelated, but I saw the warning on an otherwise clean build, so I threw it in.